libraries/Microsoft.Bot.Builder.Ai/SmartTalk.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+    public class SmartTalk
+    {
+        public const string smartTalkServiceEndPoint = "https://smarttalkmanagementservice.azure-api.net/smarttalk/api/SmartTalk/GetResponse";
+        public const string APIManagementHeader = "Ocp-Apim-Subscription-Key";
+        public const string JsonMimeType = "application/json";
+
+        private static HttpClient g_httpClient = new HttpClient();
+        private readonly HttpClient _httpClient;
+        private readonly SmartTalkMiddlewareOptions _options;
+
+        public SmartTalk(SmartTalkMiddlewareOptions options, HttpClient httpClient = null)
+        {
+            _httpClient = httpClient ?? g_httpClient;
+            _options = options ?? throw new ArgumentNullException(nameof(options));
+        }
+
+        public async Task<SmartTalkResults> GetAnswers(string query)
```

Comment:
Answer here seems a misleading term, and makes me think more Q&A. In this case, perhaps "GetResponse" or similar?